src/Mapper/Space.php

```diff
@@ -28,18 +27,6 @@ public function map($resource, array $data)
         return $this->hydrate($resource ?: ResourceClass::class, [
             'sys' => $this->buildSystemProperties($data['sys']),
             'name' => $data['name'],
-            'locales' => \array_map([$this, 'buildLocale'], $data['locales']),
```

Comment:
Space still has locales

Reply:
Only for backwards compatibility purposes in the API, the handling of locales should happen at the environment level, from now on...